bugbug/model.py

```diff
@@ -144,8 +144,13 @@ def print_labeled_confusion_matrix(confusion_matrix, labels, is_multilabel=False
                 if table_labels[i] != "__NOT_CLASSIFIED__"
                 else "Not classified"
             )
+        if is_multilabel and test_classes is not None:
+            confusion_matrix_header.append("Not classified")
         for i in range(len(table)):
             table[i].insert(0, f"{table_labels[i]} (Actual)")
+            if is_multilabel and test_classes is not None:
+                y_true_count = (test_classes[:, num].tolist()).count(i)
+                table[i].append(y_true_count - sum(table[i][1:]))
```

Comment:
The number of "Not classified" was originally part of the table but dropped earlier:
```py
    # Don't show the Not classified row in the table output
    if "__NOT_CLASSIFIED__" in labels and not is_multilabel:
        confusion_matrix_table.pop(labels.index("__NOT_CLASSIFIED__"))
```

It may be cleaner to avoid dropping it in the first place.